pkgs/by-name/ro/rofi-network-manager/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  rofi,
+  rofi-wayland,
+  networkmanager,
+  libnotify,
+  coreutils,
+  sudo,
+  backend ? "x11",
+}:
+
+assert lib.assertOneOf "backend" backend [
+  "x11"
+  "wayland"
+];
+
+stdenv.mkDerivation {
+  pname = "rofi-network-manager";
+  version = "2024-09-03";
+
+  src = fetchFromGitHub {
+    owner = "meowrch";
+    repo = "rofi-network-manager";
+    rev = "90302dd1c0ea2d460a3455a208c10dff524469cd";
+    sha256 = "sha256-D8/Lh5a5rAUOhCXbjmL65PFzgmj3uu2mwCtxakHTefM=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    install -Dm 0755 network-manager.sh $out/bin/rofi-network-manager
+  '';
+
+  wrapperPath = lib.makeBinPath (
+    [
+      coreutils
+      libnotify
+      sudo
+      networkmanager
+    ]
+    ++ lib.optionals (backend == "x11") [
+      rofi
+    ]
+    ++ lib.optionals (backend == "wayland") [
+      rofi-wayland
+    ]
+  );
+
+  fixupPhase = ''
```

```suggestion
  postFixup = ''
```
lets not override things